eslint.config.mjs

```diff
@@ -281,33 +184,15 @@ export default tseslint.config(
             },
           ],
           groups: ['title', 'component'],
-          ...perfectionistCommonConfig.partition,
         },
       ],
-      'prefer-regex-literals': 'error',
-      radix: 'error',
-      // Start of React rules
-      'react/jsx-key': 'off',
-      'react/no-unknown-property': 'off',
-      'react/prop-types': 'off',
+      'perfectionist/sort-modules': 'off', // TODO: do we want this? In some cases, this means you use a type before it's defined (see Grid for example). That makes it harder to read imo.
```

Comment:
What do we want to do here?

Reply:
Hmm that’s too bad. I agree that using a type before it’s defined is confusing. I wonder why they sort exported members before local ones in their [default groups config](https://perfectionist.dev/rules/sort-modules.html#groups) – the other way around seems much more practical: we extract local types before using them in exported ones.

I’d advocate setting our own value for `groups`, but if you want to limit configuration here as well, let’s keep this rule turned off.